### DISCUSSION TITLE
Klipper

### DISCUSSION THREAD

**Discussion by 3D-Frank** [OrcaSlicer/OrcaSlicer#165] [Ideas]
Hi!
Is it already possible to integrate a printer such as the Anycubic Vyper (with Voron Stealthburner modification) under Klipper?

**Comment by SoftFever** [comment]
@3D-Frank Klipper printers are fully supported.
You can use any Voron/Ratrig machine as a template and then change the bed size to your printer's size.

**Reply by 3D-Frank** [reply]
Thanks for the quick and positive reply!
